webpackDevServer.js

```diff
@@ -1,15 +1,19 @@
-/* eslint no-console:0 */
-
-const WebpackDevServer = require("webpack-dev-server");
-const webpack = require("webpack");
-const webpackConfig = require("./webpack.config.js");
+// @flow
+const { targets, createConfig } = require('./webpack.common');
+const path = require('path');
 const PORT = 7936;
```

Comment:
Is there any specific reason using the port 7936?

Reply:
We wanted to avoid a collision with other services that are usually running in the normal dev environment at Khan Academy.